fancy-explosions/BangazonWorkforce
# Jm dept det

# Description

Used should be able to view department details

Fixes Issue # (6)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

    git fetch --all
    git checkout JM-dept-det
    Select Departments on the nav bar
    Select the Details tab on one a Department and verify if department details are displayed

# Checklist:
